scanner/services/indexer.go

```diff
@@ -129,8 +148,7 @@ func (s *indexerService) RegisterServiceServer(grpcServer *grpc.Server) {
 
 // AuthFuncOverride specifies the auth criteria for this API.
 func (s *indexerService) AuthFuncOverride(ctx context.Context, fullMethodName string) (context.Context, error) {
-	// TODO: Setup permissions for indexer.
-	return ctx, allow.Anonymous().Authorized(ctx, fullMethodName)
+	return ctx, indexerAuth.Authorized(ctx, fullMethodName)
```

Comment:
I'd say as a pattern it'd probably be better to have the condition of either using anonymous access or not embedded here instead of the `indexerAuth` func type that you currently have.